wled00/wled_server.cpp

```diff
@@ -39,20 +39,20 @@ void handleUpload(AsyncWebServerRequest *request, const String& filename, size_t
   if (!index) {
     String finalname = filename;
     if (finalname.charAt(0) != '/') {
-      finalname = "/" + finalname; // prepend slash if missing
+      finalname = '/' + finalname; // prepend slash if missing
     }
 
     request->_tempFile = WLED_FS.open(finalname, "w");
-    DEBUG_PRINT("Uploading ");
+    DEBUG_PRINT(F("Uploading "));
     DEBUG_PRINTLN(finalname);
-    if (finalname == "/presets.json") presetsModifiedTime = toki.second();
+    if (finalname.equals(F("/presets.json")) >= 0) presetsModifiedTime = toki.second();
```

Comment:
I think that string.equals() returns bool, so should not be tested for ">=0"

Reply:
100% correct 😅